README.md

```diff
@@ -225,9 +161,9 @@ Example:
 
 > Describe 1 - 3 features you will add to the project.
 
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+- [ ] **[coming soon]**
+- [ ] **[coming soon]**
+- [ ] **[coming soon]**
```

Comment:
- [ ] Consider removing this section as it is currently redundant and has nothing in it.